mon-pix/app/routes/application.js

```diff
@@ -35,7 +35,12 @@ export default Route.extend(ApplicationRouteMixin, {
 
   // XXX: For override the sessionInvalidated from ApplicationRouteMixin to avoid the automatic redirection
```

Comment:
Est-ce que quelqu'un se souvient pourquoi cette surcharge de `sessionInvalidated` existe ?

Est-ce que quelqu'un comprend ce commmentaire `XXX: For override the sessionInvalidated from ApplicationRouteMixin to avoid the automatic redirection` ?

Si ce n'est pas le cas, est-ce que ce ne serait pas l'occasion de, au minimum, modifier le commentaire ? Et voir si cette surcharge a encore une raison d'être ?

Reply:
Cette surcharge existe pour empêcher la redirection vers login lorsqu'on vient du GAR. Elle a encore raison d'être

Reply:
Je ne comprends pas bien. Avant cette PR, cette surcharge _faisait une transition vers `login`_, donc elle ne pouvait _pas_ avoir comme utilité de _empêcher la redirection vers `login`_ d'accord ?

Après cette PR, je veux bien croire qu'elle joue un rôle là-dedans mais ça n'explique pas pourquoi elle existait à la base, et ça ne me convainc pas qu'elle soit le bon moyen d'y arriver — compte tenu des effets néfastes qu'elle introduit.

Reply:
En fait, avant cette PR, on a dû ajouter la redirection vers login à cause de la mise à jour de ember-simple-auth. Avant-avant cette PR, c'était un override vide pour empêcher la redirection vers login